### PR TITLE
Setting Start Time on build start

### PR DESCRIPTION
**Pull request by stjohnjohnson**

### CODE REVIEW

File: plugins/webhooks/build.js [hunk 1]
```diff
@@ -42,10 +42,13 @@ module.exports = () => ({
                 if (['SUCCESS', 'FAILURE', 'ABORTED'].indexOf(status) > -1) {
                     build.meta = request.payload.meta || {};
                     build.endTime = (new Date()).toISOString();
+                } else {
+                    build.startTime = (new Date()).toISOString();
```
Comment (d2lam) [inline plugins/webhooks/build.js]: Not sure how the launcher interacts with this. Does it only hit this endpoint when the on start and notify? Are we sure it won't do it for status = `RUNNING`?

Comment (stjohnjohnson) [inline plugins/webhooks/build.js]: That's the status it sends on start.

Comment (d2lam) [inline plugins/webhooks/build.js]: oh I mean would it hit this end points multiple times or just once?

**Comment by Filbird** [comment]
👍

File: plugins/webhooks/build.js (hunk 1 above)
Comment (stjohnjohnson) [inline plugins/webhooks/build.js]: Right now it's only twice, once on start and once on end.

**Comment by d2lam** [comment]
👍  Looks good

**Pull request merged**